src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+
+        for (Ball ball : balls) {
+            System.out.println(Lottery.getRandomBall());
+        }
```

Comment:
This loop will throw a `NullPointerException` because the `balls` array is not populated with `Ball` objects. Initialize each element of the `balls` array with `Lottery.getRandomBall()` before this loop.